tests/baselines/reference/showConfig/Shows tsconfig for single option/noDtsResolution/tsconfig.json

```diff
@@ -1,5 +0,0 @@
-{
-    "compilerOptions": {
```

Comment:
Why did this get deleted?

Reply:
I'm pretty sure this is an orphaned baseline. I have no idea how I decided to delete it though.